apps/els_lsp/src/els_diagnostics_utils.erl

```diff
@@ -31,28 +31,23 @@ included_uris(Document) ->
 dependencies([], Acc, _AlreadyProcessed) ->
   Acc;
 dependencies([Uri|Uris], Acc, AlreadyProcessed) ->
-  case els_dt_document:lookup(Uri) of
-    {ok, [Document]} ->
-      Behaviours = els_dt_document:pois(Document, [behaviour]),
-      ParseTransforms = els_dt_document:pois(Document, [parse_transform]),
-      IncludedUris = included_uris(Document),
-      FilteredIncludedUris = exclude_already_processed( IncludedUris
-                                                      , AlreadyProcessed
-                                                      ),
-      PTUris = lists:usort(
-                 lists:flatten(
-                   [pt_deps(Id) || #{id := Id} <- ParseTransforms])),
-      FilteredPTUris = exclude_already_processed( PTUris
-                                                , AlreadyProcessed
-                                                ),
-      dependencies( Uris ++ FilteredIncludedUris ++ FilteredPTUris
-                  , Acc ++ [Id || #{id := Id} <- Behaviours ++ ParseTransforms]
-                    ++ [els_uri:module(FPTUri) || FPTUri <- FilteredPTUris]
-                  , sets:add_element(Uri, AlreadyProcessed));
-    Error ->
-      ?LOG_INFO("Lookup failed [Error=~p]", [Error]),
-      []
-  end.
+  {ok, Document} = els_utils:lookup_document(Uri),
```

Comment:
Same here. If a project is misconfigured, some dependencies will not be found and a hard pattern match here would result in a less stable server. Maybe we want to modify the `els_utils:lookup_document/1` so that if it fails after indexing it returns an error and keep the case in the callers?